gcloud-java-compute/src/main/java/com/google/gcloud/compute/ImageInfo.java

```diff
@@ -393,7 +393,7 @@ Image toPb() {
   /**
    * Returns a builder for an {@code ImageInfo} object given the image identity and an image
    * configuration. Use {@link DiskImageConfiguration} to create an image from an existing disk. Use
-   * {@link RawImageConfiguration} to create an image from a raw image stored in Google Cloud
+   * {@link StorageImageConfiguration} to create an image from a raw image stored in Google Cloud
```

Comment:
Fine as is but I do wonder if we should really say "from a raw image". I assume the "raw" part depends on the `sourceType` which indeed `RAW` is its only option but was not sure if that needs to slip into here.